staff/ericdarbra/css/mediaquery/styles/desktop.css

```diff
@@ -0,0 +1,59 @@
+* {
```

Comment:
TIP some css rules repeat in mobile and desktop. may you group them in main.css?